draft-ietf-ppm-dap.md

```diff
@@ -1141,6 +1144,13 @@ VDAF types:
 * PrepShare
 * PrepMessage
 
+Additionally, to prevent cross-VDAF confusion attacks, in which an input
+intended for a particular VDAF configuration is evaluated under another VDAF
```

Comment:
```suggestion
Additionally, to prevent cross-VDAF confusion attacks, in which a message
intended for a particular VDAF configuration is evaluated under another VDAF
```